requirements.txt

```diff
@@ -1,8 +1,12 @@
 async_lru>=1.0,<2.1
+ansible-sign==0.1.1
 galaxy_importer>=0.4.5,<0.5
 GitPython>=3.1.24,<3.2
 jsonschema>=4.9,<4.20
 Pillow>=7.0,<10.2
 pulpcore>=3.25.0,<3.40
 PyYAML>=5.4.1,<7.0
 semantic_version>=2.9,<2.11
+setuptools>=39.2,<66.2.0
```

Comment:
What is `setuptools` needed for?

Reply:
I am not sure anymore why I added it here, so removing it.